airflow/cli/commands/variable_command.py

```diff
@@ -76,7 +78,29 @@ def variables_import(args):
 
 def variables_export(args):
     """Exports all the variables to the file."""
-    _variable_export_helper(args.file)
+    var_dict = {}
+    with create_session() as session:
+        qry = session.query(Variable).all()
```

Comment:
Can we please use more meaningful variable names instead of qry, var, val?

Reply:
Fair point. I just refactored the helper function into this. Will update the patch.